smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -157,15 +157,23 @@ public Task Install()
             // Use apt-get if a package name is given, or dpkg if a package file is given.
             // We'd like to use apt-get for both cases, but older versions of apt-get (e.g.,
             // in Raspbian) can't accept a package file.
-            if (string.IsNullOrEmpty(this.archivePath))
+            if (System.Environment.Is64BitOperatingSystem)
```

Comment:
Why are you checking for 64-bit OS here? And can you add a code comment to explain the rationale?

Also, this could be written more succinctly like this:
```csharp
if (System.Environment.Is64BitOperatingSystem || string.IsNullOrEmpty(this.archivePath))
{
    commandName = "apt-get";
    commandArgs = $"--yes install {PackageName}";
}
else
{
    commandName = "dpkg";
    commandArgs = $"--force-confnew -i {this.archivePath}";
}
```